front-app/src/views/Products.tsx

```diff
@@ -0,0 +1,188 @@
+  };
+
+  if (isLoading) return <div>Loading...</div>;
+
+  return (
+    <section className='flex flex-col relative gap-16 w-full'>
+      <TopNav>
+        <input
+          type='text'
+          value={textValue}
+          onChange={handleInputChange}
+          placeholder=''
+          className='bg-gray-100 px-12 md:px-6 h-full rounded-3xl w-full border-radius border-solid border-black'
+        ></input>
+        <button
+          onClick={handleInputClick}
+          className='absolute bg-[#9BC7FF] rounded-full w-8 h-8 right-[93%] md:right-[1%] translate-y-[7%]'
+        >
+          <img src={SearchIcon} className='w-20 h-4'></img>
+        </button>
+        <div className='flex md:hidden'>
+          <img src={Vector} className='w-8 h-8 absolute right-[1%] translate-y-[7%]'></img>
+        </div>
+      </TopNav>
+      <div className='bg-gray-100'>
+        <div className='w-[80%] m-inline flex flex-col'>
+          <div className='flex flex-col mt-10 md:mt-20 gap-1'>
+            <h6 className='hidden md:flex uppercase text-xl font-normal'>regalos</h6>
+            <h6 className='hidden md:flex uppercase text-4xl font-medium'>Food Box</h6>
+          </div>
+          <div className='flex md:justify-start justify-center'>
+            {totalResults && totalResults > 1 ? (
+              <p className='hidden md:flex font-normal text-lg text-gray-500 mt-3'>
+                Más de {totalResults - 1} opciones
+              </p>
+            ) : (
+              <p className='hidden md:flex font-normal text-lg text-gray-500 mt-3'>{totalResults} opcion</p>
+            )}
+            {totalResults && (
+              <p className='md:hidden flex font-normal text-lg text-gray-500 mt-3 text-center'>
+                {totalResults} resultados
+              </p>
+            )}
+          </div>
+
+          <div className='hidden lg:flex items-center justify-between'>
+            <div className='flex gap-4 mt-10 text-base'>
+              {categoriesData?.map((category) => (
+                <button
+                  onClick={() => handleCategoryClick(category.id)}
+                  className={
+                    categoryFilters.includes(category.id)
+                      ? 'border-solid border-black bg-gray-600 text-white border-[1px] rounded-md py-1.5 px-5 cursor-pointer'
+                      : 'border-solid border-black border-[1px] rounded-md py-1.5 px-5 cursor-pointer'
+                  }
+                >
+                  {category.name}
+                </button>
+              ))}
+            </div>
+            <div>
+              <input className='bg-transparent' list='order' />
+              <select className='bg-transparent h-[40px]' id='order' value={querySortBy} onChange={handleSortBy}>
+                <option value=''>Ordenar por</option>
+                <option value='price_asc'>Precio ascendente</option>
+                <option value='price_desc'>Precio descendente</option>
```

Comment:
Let's refactor this. Ordenar por is not an option. The only two options here should be `Precio ascendente` and `Precio descendente`, the `Ordenar por` should be displayed separately. Also let's add some styles to the select so it matches the look and feel of the hole page